### DISCUSSION TITLE
How to translate chatify?

### DISCUSSION THREAD

**Discussion by juliaokataleko** [munafio/chatify#371] [General]
I'm a portugues native speaker and i need to translate to portuguese som words. Iam not seeing where to do that!

**Comment by betonaranjof988** [comment]
Hi @juliaokataleko , can you resolved this? or how to did if you can translated?

**Reply by munafio** [reply]
there are no translation files but you can modify the views to your needs

**Reply by betonaranjof988** [reply]
thanks for reply. :)
